src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -3,11 +3,11 @@
 --********************************************************************/
 
 using System;
+using System.Collections.Generic;
```

Comment:
The file is not sorted but I agree to merge "as is" - if we'll have follow formatting PR.

Reply:
@iSazonov I'm confused. This file is sorted. What is out of order?

Reply:
Line 67.

Reply:
ah. that one can probably be fixed with automatic property `{ get; private set }`, which is on the checklist already.

Reply:
@markekraus Pow("Please", 10) make the follow PRs as small as possible - split work by file/type formattings. :-)

Reply:
@iSazonov if you have ideas on how the work should be split, please update #5610. I still don't have a good handle on what is too big or too small and, honestly, sometimes the feedback on that is inconsistent.

Reply:
Maybe interesting https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/